compiler/rustc_ast_lowering/src/lib.rs

```diff
@@ -875,11 +861,7 @@ impl<'a, 'hir> LoweringContext<'a, 'hir> {
                 // ```
                 //
                 // In both cases, we don't want to synthesize any tokens
-                MacArgs::Delimited(
-                    dspan,
-                    delim,
-                    self.lower_token_stream(tokens.clone(), CanSynthesizeMissingTokens::No),
-                )
+                MacArgs::Delimited(dspan, delim, self.lower_token_stream(tokens))
```

Comment:
```suggestion
                MacArgs::Delimited(dspan, delim, tokens.flattened())
```
as `lower_token_stream` has no other usage?